.github/workflows/ruby.yml

```diff
@@ -24,6 +24,7 @@ jobs:
             gemfile: rails_5_2
     env:
       BUNDLE_GEMFILE: gemfiles/${{ matrix.gemfile }}.gemfile
+      CI: '1'
```

Comment:
Apparently it's already set to `true`: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables maybe we could just check for its presence?

Reply:
Oh nice, let's use that!